scripts/docker/jlink/license.txt

```diff
@@ -0,0 +1,11 @@
+By accessing and using Software and Materials provided by SEGGER as free download, you acknowledge and agree to the following Terms of Use. If you do not agree to these Terms, do not download or use any Software or Material.
```

Comment:
Do we need to have this file in repo? We need to keep it up to date with actual license? Would it be possible to download license only into docker image (Dockerfile has recipe to download) and cat if ACCEPT_JLINK_LICENSE=1 ?

Reply:
Please check response https://github.com/nrfconnect/sdk-nrf/pull/16290#discussion_r1662476217